mdal/api/mdal.h

```diff
@@ -98,11 +99,18 @@ MDAL_EXPORT const char *MDAL_Version();
 //! Returns last status message
 MDAL_EXPORT MDAL_Status MDAL_LastStatus();
 
-//! Set callback for logging output
-//! if callback = nullptr avoids logs
-//! TODO: say that it defaultly outputs to stdout
+//! Set custom callback for logging output. By default output is not being shown anywhere,
```

Comment:
use /**
*
*/ for multiline comments

add new line after first sentence (first line is brief comment, other lines are long descriptions)

- use 3rd person for docs (Sets, Returns, ...)